include/openenclave/enclave.h

```diff
@@ -641,6 +641,45 @@ oe_result_t oe_generate_attestation_certificate(
     uint8_t** output_cert,
     size_t* output_cert_size);
 
+/**
+ * oe_generate_attestation_certificate_v2
+ *
+ * Similar to oe_generate_attestation_certificate
+ * This function generates a self-signed x.509 certificate with an embedded
+ * quote from the underlying enclave. It also allows a user to pass in
+ * expiration date for the certificate.
+ *
+ * @param[in] subject_name a string contains an X.509 distinguished
+ * name (DN) for customizing the generated certificate. This name is also used
+ * as the issuer name because this is a self-signed certificate
+ * See RFC5280 (https://tools.ietf.org/html/rfc5280) specification for details
+ * Example value "CN=Open Enclave SDK,O=OESDK TLS,C=US"
+ *
+ * @param[in] private_key a private key used to sign this certificate
+ * @param[in] private_key_size The size of the private_key buffer
```

Comment:
```suggestion
 * @param[in] private_key_size the size of the private_key buffer
```
nit: rest of file doesn't seem to use punctuation at end and doesn't seem to use upper case to start.  Doesn't matter either way to me, as long as its consistent.